### DISCUSSION TITLE
Tags/search option for undetermined specimens

### DISCUSSION THREAD

**Discussion by EMTuckerLab** [Symbiota/community-board#126] [Wish List]
It would be incredibly helpful if there were a way to tag and search for specimen records that are unidentified. Preferably at multiple hierarchal levels (i.e., only identified to genus/family/ordinal level etc.).  It would make it much easier to improve the identifications, as unidentified records could be shared with willing taxonomists to identify. Ideally there would also be a way for identifiers to tag or make comments as to what the specimen is so that we can update our databases... Thanks!

**Comment by themerekat** [comment]
Thanks for your feedback! Right now you can share a link that displays all specimens for which scientific name is empty/null using a link generated like this: 
![image](https://user-images.githubusercontent.com/16688784/158826046-a6e58ff3-daca-4380-99ea-d1e4177323b5.png)
However, this can only be shared with someone who has editing access to the collection.

Identifiers can make comments on individual specimens using the Comments tab, as long as they are logged in to a user account (they don't need to have editing permissions in this case). More info on that option here: https://biokic.github.io/symbiota-docs/user/comment/

I think it would be neat to be able to search for specimens that are only ID'ed to genus/family. It would also be neat to be able to search for specimens that have no value in the scientific name field. I don't think you can do that from the public search right now.

**Comment by themerekat** [comment]
Interestingly, a user of the CCH2 portal just asked about this possibility as well. I've added his comments to the wish list item that I created: https://github.com/BioKIC/symbiota-docs/issues/128
